src/types/markers.js

```diff
@@ -269,9 +269,20 @@ export type NetworkPayload = {
   type: 'Network',
   URI?: string,
   id: number,
+  pri: number,
+  count: number,
   status: string,
   startTime: Milliseconds,
   endTime: Milliseconds,
+  domainLookupStart?: Milliseconds,
+  domainLookupEnd?: Milliseconds,
+  connectStart?: Milliseconds,
+  tcpConnectEnd?: Milliseconds,
+  secureConnectionStart?: Milliseconds,
+  connectEnd?: Milliseconds,
+  requestStart?: Milliseconds,
+  responseStart?: Milliseconds,
+  responseEnd?: Milliseconds,
```

Comment:
> I see these new properties are optional. Are these new properties always present for some status' values, and always missing otherwise ?

While they're always tied to a status today, I can't guarantee that they will always be, and also when I then add any new items I'll need to use ? since old profiles won't have them (yes, I could add more code to convert old profiles, but this seems overly-complex and more prone to error, plus it doesn't help with ones I remove in the future)

Reply:
Converting old profiles wouldn't be useful here anyway, as we can't invent data we don't have :-)

OK, thanks for the explanation, then it's good for me.